sys/net/gnrc/network_layer/ipv6/gnrc_ipv6.c

```diff
@@ -742,13 +745,19 @@ static void _receive(gnrc_pktsnip_t *pkt)
 #ifdef MODULE_GNRC_IPV6_WHITELIST
         if (!gnrc_ipv6_whitelisted(&((ipv6_hdr_t *)(pkt->data))->src)) {
             DEBUG("ipv6: Source address not whitelisted, dropping packet\n");
+#ifdef MODULE_GNRC_ICMPV6_ERROR
+            gnrc_icmpv6_error_dst_unr_send(ICMPV6_ERROR_DST_UNR_PROHIB, pkt);
+#endif
```

Comment:
Does it make sense to provide a single stub for `gnrc_icmpv6_error_dst_unr_send()` when `MODULE_GNRC_ICMPV6_ERROR` is not included? then you don't have to put `ifdefs` around all the occurrences in the code here.

Reply:
Mostly to improve readability by reducing the number of `ifdef` statements in the code.

Reply:
Makes sense! Will do.

Reply:
Addressed.